charts/k8ssandra/values.yaml

```diff
@@ -173,6 +173,27 @@ cassandra:
       tag: 6c64f9c4
       # -- Pull policy for the system-logger image
       pullPolicy: IfNotPresent
+  # -- cassandraYaml directly configures properties in cassandra.yaml.
+  cassandraYaml:
+    # concurrent_reads:
```

Comment:
How did you decide what properties "made the cut" so to speak to be supported and what didn't?  Any methodology to establishing what was more important?

Reply:
For the most part I just included those requested in #1023 and #1166, so an ask and ye shall receive methodology :)

Actually, I excluded a few that will be a bit more involved. They will require configuring volumes, like the commit log directory. I definitely think we want to expose these, but I am just focusing low hanging fruit right now.